pkg/aws/service.go

```diff
@@ -233,6 +232,17 @@ func Construct(cfg Config) (types.Service, error) {
 			return []delegation.Option{delegation.WithExpiration(int(time.Now().Add(time.Hour).Unix()))}
 		},
 	)
+	blockIndexCfg := cfg.Config.Copy()
+	blockIndexCfg.Region = cfg.LegacyBlockIndexTableRegion
+	legacyBlockIndexStore := NewDynamoProviderBlockIndexTable(dynamodb.NewFromConfig(blockIndexCfg), cfg.LegacyBlockIndexTableName)
+
+	// allow claims synthethized from the block index table to live a bit longer after they are expired in the cache
+	// so that the service doesn't return cached but expired delegations
+	synthetizedClaimExp := redis.DefaultExpire + 10*time.Minute
```

Comment:
Blocking: the expiration time on the claims cache is configurable and is configured much greater in prod (1 week) than redis.DefaultExpire (1 hour). Please update (otherwise claims will expire before they expire from cache)

```suggestion
	synthetizedClaimExp := time.Duration(cfg.ClaimsCacheExpirationSeconds)*time.Second + 10*time.Minute
```

Reply:
didn't realize there was a setting for that, thanks!

I completely made up the 10 min buffer, does it make sense? I guess the claim will be used immediately by the gateway and 10 minutes is more than enough for it to complete the operation, right? Are there any other use cases that may need the claim to be valid for longer?